packages/std/src/math/decimal.rs

```diff
@@ -182,6 +182,14 @@ impl ops::Mul<Uint128> for Decimal {
     }
 }
 
+impl ops::Div<Uint128> for Decimal {
+    type Output = Self;
+
+    fn div(self, rhs: Uint128) -> Self::Output {
+        Decimal(self.0 / rhs.u128())
+    }
+}
```

Comment:
Nice. Could you also do DivAssign while you're here?

Reply:
Yep!